src/app/static/src/app/store/baseline/actions.ts

```diff
@@ -25,9 +24,7 @@ export const actions: ActionTree<BaselineState, RootState> & BaselineActions = {
     async getBaselineData({commit}) {
         await api<BaselineActionTypes, BaselineErrorActionTypes>(commit)
             .ignoreErrors()
-            .withSuccess("BaselineDataLoaded")
-            .get<BaselineData>("/baseline/");
+            .withSuccess("PJNZLoaded")
+            .get<PjnzResponse>("/baseline/pjnz/");
```

Comment:
Updated to return only the pjnz response. In the next PR which adds the shape file stuff to the front end will be getting the shape file here too.